oak_functions/examples/weather_lookup/README.md

```diff
@@ -0,0 +1,41 @@
+# Oak functions `weather_lookup` example
+
+This is an Oak Functions application that expects to receive geographic
+coordinates (latitude and longitude) in the request, and responds with the
+weather at that location, by looking it up in a lookup data set that is
+periodically refreshed from an external source.
+
+## Running manually
+
+To build and run this example manually follow these steps:
+
+1. Build the weather lookup Wasm module:
+
+   ```shell
+   cargo -Zunstable-options build --release \
+       --target=wasm32-unknown-unknown \
+       --manifest-path=./oak_functions/examples/weather_lookup/module/Cargo.toml \
+       --out-dir=./oak_functions/examples/weather_lookup/bin
+   ```
+
+1. Build `oak_functions_loader` (with debugging enabled if needed):
+
+   ```shell
+   cargo build --manifest-path=./oak_functions/loader/Cargo.toml \
+       --release \
+       --features=oak-unsafe
```

Comment:
Why do we need `unsafe`?
Is it to work with Wasm modules?

Reply:
Just to have some logging I guess, it was already there.

Reply:
Yes. It is for logging. We have to removed it for the prod version, I think.